data/apply-planet_australia_suburb_recast.sql

```diff
@@ -0,0 +1,31 @@
+-- Suburbs in Australia are treated more like cities, so this recasts them to  place=town
+
+with bbox as
+         (select *
+          from (values
+                    (st_transform(st_geomfromtext('Polygon ((116.16 -14.31, 104.29 -22.95, 113.34 -38.30, 155.33 -47.28, 156.30 -21.83, 144.91 -9.41, 127.14 -9.73, 116.16 -14.31))', 4326), 3857))
+               ) as b (geom)),
+
+     suburb as
+         (select *
+          from planet_osm_point l where tags->'place'='suburb'),
+
+     selection as
+         (select * from suburb,
+                        bbox
+          where st_within(way, bbox.geom)),
+
+    trim_major_cities as
```

Comment:
Explain that these are already there are cities, so we're trimming so we don't end up with duplicate labels on the map.